packages/x-charts-vendor/lib-vendor/d3-array/src/disjoint.js

```diff
@@ -11,10 +11,10 @@ function disjoint(values, other) {
   for (const v of values) {
     if (set.has(v)) return false;
     let value, done;
-    while (({
+    while ({
```

Comment:
These changes make sense to me.
Looks like there was one extra parenthesis (`(...)`) before this babel bump.
Bump asking for a second pair of eyes @mui/xcharts

Reply:
I agree I assume it's due to this polish introduced in v7.25.4 https://github.com/babel/babel/pull/16722